pkgs/development/python-modules/logica/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  duckdb,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "logica";
+  version = "1.3.14159265358";
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-BC3pjy5jde/2MWvR2oMbli0C4GssV7hdIQ82Hpc2RAM=";
+  };
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace-fail '"logica/README.md"' '"README.md"'
+  '';
+
+  build-system = [ setuptools ];
+
+  dependencies = [ duckdb ];
+
+  pythonImportsCheck = [ "logica" ];
+
+  meta = {
+    description = "Declarative logic programming language for data manipulation";
+    homepage = "https://logica.dev/";
```

Comment:
add changelog

Reply:
I couldn't find a changelog for this project

Reply:
I can't either. No problem, thanks.